### PR TITLE
store revision list in `keyPrefix:revisions`

### LINKED ISSUE #28: Don't use prefix as key name for revisions list
It's confusing that the plugin uses the prefix as the key name for the revisions list. One expects a "prefix" to be before something. Let's change it to prefix + ":revisions"

### PR DESCRIPTION
**Pull request by ghedamat**
fix #28

### CODE REVIEW

File: tests/helpers/fake-redis-client.js [hunk 1]
```diff
@@ -11,10 +12,12 @@ module.exports = CoreObject.extend({
   set: function() { },
   del: function() { },
   zadd: function(key, score , tag) {
-    this.recentRevisions.push(key + ':' + tag);
+    assert(key.match(/:revisions$/));
```
Comment (ghedamat) [inline tests/helpers/fake-redis-client.js]: @achambers @lukemelia I'm doing an assertion for this in the fake client.. it's not super nice and not even needed bug given that we stub redis entirely it seemed the most appropriate place...

Comment (lukemelia) [inline tests/helpers/fake-redis-client.js]: yeah, this is kind of ugly. maybe you can extend the fake client with a method that has the assert for just the test that cares about it?

Comment (ghedamat) [inline tests/helpers/fake-redis-client.js]: yeah, as mentioned, I agree..
it spans across a bunch of tests, but we already reopen the redis client in some tests anyway so it should do no harm..

will ping you shortly with an update

**Comment by ghedamat** [comment]
@lukemelia ping, let me know if this sounds better

File: tests/helpers/fake-redis-client.js [hunk 2]
```diff
@@ -11,10 +11,11 @@ module.exports = CoreObject.extend({
   set: function() { },
   del: function() { },
   zadd: function(key, score , tag) {
-    this.recentRevisions.push(key + ':' + tag);
+    var prefix = key.replace(':revisions','');
+    this.recentRevisions.push(prefix + ':' + tag);
   },
-  zrem: function(val,revision) {
-    var i = this.recentRevisions.indexOf(revision)
+  zrem: function(key,revision) {
```
Comment (lukemelia) [inline tests/helpers/fake-redis-client.js]: add space after comma

**Comment by lukemelia** [comment]
One tiny nit, otherwise, looks good!

**Comment by ghedamat** [comment]
@lukemelia done!

**Comment by ghedamat** [comment]
also updated the README since we know list the keys there as well..

**Comment by lukemelia** [comment]
:+1:

**Pull request merged**